tests/func-tests/node_placement_test.go

```diff
@@ -36,28 +36,30 @@ var _ = Describe("[rfe_id:4356][crit:medium][vendor:cnv-qe@redhat.com][level:sys
 		originalWorkloadSpec hcov1beta1.HyperConvergedConfig
 		cli                  client.Client
 		cliSet               *kubernetes.Clientset
+		workerNodes          *v1.NodeList
 	)
 
 	BeforeAll(func(ctx context.Context) {
 		cli = tests.GetControllerRuntimeClient()
 		cliSet = tests.GetK8sClientSet()
 
-		nodes := listNodesByLabels(ctx, cliSet, "node-role.kubernetes.io/control-plane!=")
-		tests.FailIfSingleNode(len(nodes.Items) < 2)
+		workerNodes = listNodesByLabels(ctx, cliSet, "node-role.kubernetes.io/worker")
+		//workerNodes = listNodesByLabels(ctx, cliSet, "node-role.kubernetes.io/control-plane!=")
```

Comment:
is there a reason to keep this commented out?

(in a compact cluster, all 3 nodes have both `node-role.kubernetes.io/worker` and `node-role.kubernetes.io/control-plane` labels.)